content/zh/docs/reference/access-authn-authz/rbac.md

```diff
@@ -2049,26 +2318,26 @@ RBAC 鉴权组件尝试对该 API 请求鉴权。如果 RBAC 也拒绝了该 API
 鉴权组件。这意味着被 RBAC 或 ABAC 策略所允许的任何请求都是被允许的请求。
 
 <!--
-When the apiserver is run with a log level of 5 or higher for the RBAC component
-(`--vmodule=rbac*=5` or `--v=5`), you can see RBAC details in the apiserver log
-(prefixed with `RBAC:`).
+When the kube-apiserver is run with a log level of 5 or higher for the RBAC component
+(`--vmodule=rbac*=5` or `--v=5`), you can see RBAC denials in the API server log
+(prefixed with `RBAC`).
 You can use that information to determine which roles need to be granted to which users, groups, or service accounts.
 -->
-如果 API 服务器启动时，RBAC 组件的日志级别为 5 或更高（`--vmodule=rbac*=5` 或 `--v=5`），
-你可以在 API 服务器的日志中看到 RBAC 的细节 （前缀 `RBAC:`）
+如果 kube-apiserver 启动时，RBAC 组件的日志级别为 5 或更高（`--vmodule=rbac*=5` 或 `--v=5`），
+你可以在 API 服务器的日志中看到 RBAC  拒绝的细节（前缀 `RBAC`）
 你可以使用这些信息来确定需要将哪些角色授予哪些用户、组或服务帐户。
 
 <!--
 Once you have [granted roles to service accounts](#service-account-permissions) and workloads
 are running with no RBAC denial messages in the server logs, you can remove the ABAC authorizer.
 -->
-一旦你[将角色授予服务账户](#service-account-permissions) ，工作负载运行时
-在服务器日志中没有出现 RBAC 拒绝消息，就可以删除 ABAC 鉴权器。
+一旦你[将角色授予服务账户](#service-account-permissions)且工作负载运行时，
+服务器日志中没有出现 RBAC 拒绝消息，就可以删除 ABAC 鉴权器。
 
 <!--
-### Permissive RBAC Permissions
+### Permissive RBAC permissions
 
-You can replicate a permissive policy using RBAC role bindings.
+You can replicate a permissive ABAC policy using RBAC role bindings.
```

Comment:
Please update the localization of this line.

Reply:
How about this wording?
> 你可以使用 RBAC 角色绑定复制宽松的 ABAC 策略。

Reply:
LGTM,Thanks.